asmcli/tests/lib/util.bats

```diff
@@ -7,6 +7,45 @@ setup() {
   context_init
 }
 
+@test "UTIL: Fleet API should work with compatible settings" {
+  run context_set-option "EXPLICIT_FLEET_API" "1"
+  run context_set-option "CA" "mesh_ca"
+  GKE_RELEASE_CHANNEL="regular"
+  CHANNEL="regular"
+  assert use_fleet_api
+}
+
+@test "UTIL: Fleet API should only work when requested" {
+  run context_set-option "CA" "mesh_ca"
+  GKE_RELEASE_CHANNEL="regular"
+  CHANNEL="regular"
+  refute use_fleet_api
+}
+
+@test "UTIL: Fleet API should not work with stable channel" {
```

Comment:
This is a guard for a feature (that I can't remember now, maybe VPC-SC?) that hasn't finished rollout yet in stable and I heard might have a behavior difference? It will go away in O(weeks).

Reply:
Ah okay, yes, VPC-SC.